master/getting-started/docker/installation/vagrant-coreos/index.md

```diff
@@ -61,7 +61,7 @@ And finally check that Docker is running on both hosts by running
 
 ## 2. Try out Calico networking
 
-Now you have a basic two node CoreOS cluster setup, see the [Calico with Docker Networking Walkthrough]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/basic) for information on how to try Calico Networking.
+Now you have a basic two node CoreOS cluster setup, see the [Calico with Docker Networking Walkthrough]({{site.baseurl}}/{{page.version}}/getting-started/docker/installation/manual) for information on how to install Calico Networking.
```

Comment:
OK but this is still odd though.

Firstly is isn;t the "Calico with Docker Networking Walkthrough" - it's the installation steps.  It feels very odd to be directed back to a manual install guide after doing a vagrant install guide.  Maybe the install and the launch need to be different docs or different sections that we refer to.